core/variant/variant_utility.cpp

```diff
@@ -506,6 +506,33 @@ struct VariantUtilityFunctions {
 		r_error.error = Callable::CallError::CALL_OK;
 	}
 
+	static inline void print_verbose(const Variant **p_args, int p_arg_count, Callable::CallError &r_error) {
+		if (p_arg_count < 1) {
+			r_error.error = Callable::CallError::CALL_ERROR_TOO_FEW_ARGUMENTS;
+			r_error.argument = 1;
+		}
```

Comment:
Shouldn't this return if an error is found?

Reply:
Now that think of it, there's not really a reason to give an error when no arguments are provided. `print()` can be used to print a single newline.

Reply:
I agree with that too – in fact, it already seems to work that way in GDScript.